woocommerce/wc-admin
# Remove erroneous navigation component

This component must have snuck back in during a rebase.  This PR removes it in favor of the separate navigation project.

### Detailed test instructions:

1. Start up WCA.
1. Make sure no errors occur.
